src/mpl/README.md

```diff
@@ -1,44 +1,86 @@
 # Macro Placement
 
-ParquetFP-based macro cell placer, "TritonMacroPlacer".
+TritonMacroPlacer is a ParquetFP-based macro cell placer.
 The macro placer places macros/blocks honoring halos, channels
 and cell row "snapping".
 Run `global_placement` before macro placement.
 
-Approximately ceil((#macros/3)^(3/2)) sets corresponding to quadrisections
+Approximately $\Bigl\lceil [{\frac{numMacros}{3}}]^{1.5} \Bigr\rceil$ quadrisections
 of the initial placed mixed-size layout are explored and packed using
 ParquetFP-based annealing. The best resulting floorplan according to a
 heuristic evaluation function is kept.
 
 ## Commands
 
+```{note}
+- Parameters in square brackets `[-param param]` are optional.
+- Parameters without square brackets `-param2 param2` are required.
 ```
-macro_placement [-halo {halo_x halo_y}]
-                [-channel {channel_x channel_y}]
-                [-fence_region {lx ly ux uy}]
-                [-snap_layer snap_layer_number]
+
+### Triton Macro Placer
+
+```tcl
+macro_placement 
+    [-halo {halo_x halo_y}]
```

Comment:
The order looks incorrect in the `tcl` file: `-halo {vertical_width horizontal_width}`. Please fix the definition in the `tcl` file.